assets/js/base/components/notice-banner/style.scss

```diff
@@ -0,0 +1,116 @@
+.wc-block-components-notice-banner {
+	display: flex;
+	align-items: stretch;
+	align-content: flex-start;
+	color: $gray-800;
+	padding: $gap !important;
+	gap: $gap-small;
+	margin: $gap 0;
+	border-radius: 4px;
+	border-color: $gray-800;
+	font-weight: 400;
+	line-height: 1.5;
+	border: 1px solid;
+	@include font-size(small);
+	background-color: #fff;
+	box-sizing: border-box;
+
+	&:last-child {
+		margin-bottom: 0;
+	}
+
+	> .wc-block-components-notice-banner__content {
+		padding-right: $gap;
+		align-self: center;
+		white-space: normal;
+
+		> div *:first-child {
+			margin-top: 0;
+		}
+
+		> div *:last-child {
+			margin-bottom: 0;
+		}
```

Comment:
Do you know why these selectors are needed? In the examples, `.wc-block-components-notice-banner__content` doesn't seem to have a `div` inside by default.

![imatge](https://user-images.githubusercontent.com/3616980/227491516-ea8de7f1-2e0c-481b-adc7-6dd3822ac0f2.png)

Reply:
ah you're right this is no longer needed.